portfolio/src/main/webapp/elsewhere.html

```diff
@@ -7,21 +7,16 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body>
-    <div id="header">
-        <p class="header-text"> Shuli Jones </p>
-        <div id="navbar">
-            <a href='index.html' class="navlink">Home</a>
-            <a href='projects.html' class="navlink">Projects</a>
-            <a href='elsewhere.html' class="navlink">Elsewhere</a>
-        </div>
-    </div>
+    <!-- Header stored only in index.html for DRYness -->
+    <script language="javascript" type="text/javascript" src="header.txt"></script>
```

Comment:
javascript files should always have .js extension. In general you should put html in the html file as possible. I think later on you will learn how to fetch a file in javascript which you can use to read a file called header.html say and add the contents to another page.